### PR TITLE
LRQA-66294 | master

### PR DESCRIPTION
**Pull request by kyle-miho**
https://issues.liferay.com/browse/LRQA-66294

### CODE REVIEW

**Comment by kyle-miho** [comment]
This is a testcase for the bug https://issues.liferay.com/browse/LPS-88427 as well as coverage to assert that the language values are getting properly copied over when Preferences is passed in as a parameter. This is part of our Locales Test Plan
https://docs.google.com/document/d/1UdrIWd4_anL3p0TwOttBwn0kIPWLNxk0cJG22qPfCgo/edit#heading=h.s6tdya70tg54
This eventually needs to get backported all the way to 7.0.x

File: modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java [hunk 1]
```diff
@@ -199,6 +199,39 @@ public void testGetLocalizationXml() {
...
+			preferences, "test", _ENGLISH_LANGUAGE_ID, "changedValue");
+
+		String xml = LocalizationUtil.getLocalizationXmlFromPreferences(
+			preferences, portletRequest, "test", "testValue");
+
+		Assert.assertTrue(
+			"PortletPreferences not properly applied to XML",
+			xml.contains(
+				"<test language-id=\"" + _ENGLISH_LANGUAGE_ID +
+					"\">changedValue</test>"));
```
Comment (mtambara) [inline modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java:217]: @kyle-miho According to 8h of https://docs.google.com/document/d/1UdrIWd4_anL3p0TwOttBwn0kIPWLNxk0cJG22qPfCgo/edit#heading=h.s6tdya70tg54, calling getLocalizationXmlFromPreferences here is supposed to return "testValue", not "changedValue". I'm pretty sure this is a typo in the test plan, but can you confirm?

Comment (kyle-miho) [reply modules/apps/portal/portal-util-test/src/testIntegration/java/com/liferay/portal/util/test/LocalizationImplTest.java:217]: Yea thats a typo in the testplan, the way in the integration test should be the correct interaction, i updated the test plan

**Comment by mtambara** [comment]
See https://github.com/shuyangzhou/liferay-portal/pull/9944

**Pull request closed without merging**